ledger/participant-integration-api/src/test/suite/scala/platform/apiserver/services/ApiCommandServiceSpec.scala

```diff
@@ -71,6 +76,7 @@ class ApiCommandServiceSpec
         val request = SubmitAndWaitRequest.of(Some(commands))
         stub.submitAndWaitForTransactionId(request).map { response =>
           response.transactionId should be("transaction ID")
+          response.completionOffset shouldBe Some("offset")
```

Comment:
This test case makes sense. Are we also going to add tests to the Test Tool?

Reply:
Added assertions in the Test Tool as well for the command service tests